OpenRA.Mods.Common/Widgets/MapPreviewWidget.cs

```diff
@@ -83,7 +84,8 @@ public MapPreviewWidget()
 			spawnContrastColor = ChromeMetrics.Get<Color>("SpawnContrastColor");
 			spawnLabelOffset = ChromeMetrics.Get<int2>("SpawnLabelOffset");
 
-			cellWidth = Game.ModData.Manifest.TileShape == TileShape.Diamond ? 2 : 1;
+			shape = Game.ModData.Manifest.Get<MapGrid>().Type;
```

Comment:
You also need to initialize these in the other ctor.  I suspect this may be the cause of the hard breakage below.